frontend/src/components/common/form/FastSelect.tsx

```diff
@@ -117,11 +119,17 @@ export const FastSelect: React.FC<FastSelectProps> = memo(
           disabled={disabled}
           custom={custom}
           isInvalid={!!touch && !!error}
-          onBlur={handleBlur}
+          onBlur={(e: any) => {
```

Comment:
Same idea here, enforcing number type behaviour means that I don't have to format the values manually during onsubmit.

Reply:
I prefer this